Source.cpp

```diff
@@ -0,0 +1,82 @@
+#include <iostream>
+
+using namespace std;
+
+/*!
+ * Функция инициализации массива
+ */
+void create(char* array, int size);
+
+/*!
+ * Функция ввывода массива на экран
+ */
+void print(char* array, int size);
+
+/*!
+ * Функция вычисления среднего арифметического элементов массива
+ */
+double getAverage(char* array, int size);
+
+/*!
+ * Функция вывода среднего арифметического элементов массива на экран
+ */
+void print_result(char* array, int size);
+
+int main()
+{
+	setlocale(0, "");
+	int size = 1;
+	char* array;
+	cout << "Введите размер масива : ";
+	cin >> size;
+	if (size < 1)
+	{
+		cout << "Размер масива доджен быть больше 1!\n";
+		cout << "Размер масива изменен на 2!\n";
+		size = 2;
+	}
+	create(&array, size);
+	print_result(array, size);
+	system("pause");
+}
+
+void create(char* array, int size)
```

Comment:
Если параметры функции не изменяются, то они явно должны быть обозначены `const`
[By Alex Allain](https://www.cprogramming.com/tutorial/const_correctness.html)
>The const keyword allows you to specify whether or not a variable is modifiable. You can use const to prevent modifications to variables and const pointers and const references prevent changing the data pointed to (or referenced).
>Const дает вам возможность более четко документировать вашу программу и фактически обеспечивать соблюдение этой документации. Ключевое слово const обеспечивает вашим пользователям гарантии, позволяющие оптимизировать производительность без угрозы повреждения их данных, обеспечивая соблюдение вашей документации. Например, константные ссылки позволяют указать, что упомянутые данные не будут изменены; это означает, что вы можете использовать константные ссылки как простой и немедленный способ повышения производительности для любой функции, которая в настоящее время принимает объекты по значению, не беспокоясь о том, что ваша функция может изменить данные. Даже если это произойдет, компилятор предотвратит компиляцию кода и предупредит вас о проблеме. С другой стороны, если бы вы не использовали константные ссылки, у вас не было бы простого способа гарантировать, что ваши данные не были изменены.